scripts/docker/centos/6/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM centos:6
+
+RUN yum install -y cmake3 git gcc-c++
```

Comment:
Great to have centos, too! :100:

Reply:
actually I must have accidentally added the file in a commit. Centos6 has an too old kernel to be dockerized. While I did plan to add them they should not be part of the PR and I will delete it.